CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Change Log
 
+## 1.13.0 (April 10, 2024)
+
+ * [#4045](https://github.com/redhat-developer/vscode-openshift-tools/issues/4045) Can't start crc via the extension.
+ * [#4013](https://github.com/redhat-developer/vscode-openshift-tools/issues/4013) Provide an option to delete the Pod like k8s.
+ * [#4004](https://github.com/redhat-developer/vscode-openshift-tools/issues/4004) Survey doesn't work.
```

Comment:
Rename to "fix survey"

Reply:
Use Mohit's suggestion

Reply:
Used Mohit's version: `Fix Extension Survey form`